src/lib/NodeHighlighter.ts

```diff
@@ -0,0 +1,220 @@
+import { DataAttribute, dataToDatasetAttributeName, getDataAttributesFromEventPath } from '../utils/dataAttributes';
+import { EventManager } from './EventManager';
+import { HighlightRenderer, IRenderer } from './HighlightRenderer';
+import { IElementClickMessageData } from './IFrameCommunicator';
+
+export const IntersectionThreshold = 0.95;
```

Comment:
Unclear from the name what it means (what intersection?), it should either be more verbose in the name or explained with a comment